storage/innobase/xtrabackup/doc/source/installation/compiling_xtrabackup.rst

```diff
@@ -31,37 +31,53 @@ In Debian-based distributions, you need to:
 
 .. code-block:: bash
 
- $ apt-get install build-essential flex bison automake autoconf \
-    libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
-    libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
+   $ apt install build-essential flex bison automake autoconf \
+   libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
+   libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
 
-
-In ``RPM``-based distributions, you need to:
+In ``RPM``-based distributions, you need to install and enable `the
+Developer Toolset 7
+<https://www.softwarecollections.org/en/scls/rhscl/devtoolset-7/>`_ to
+make sure that you use the latest GCC compiler and development tools. |Percona Xtrabackup|
+uses GCC version 5.3 or higher. Then, install ``cmake3`` and other dependencies:
 
 .. code-block:: bash
 
-  $ yum install cmake gcc gcc-c++ libaio libaio-devel automake autoconf \
-    bison libtool ncurses-devel libgcrypt-devel libev-devel libcurl-devel \
-    vim-common
+   $ yum install cmake3 openssl-devel libaio libaio-devel automake autoconf \
+   bison libtool ncurses-devel libgcrypt-devel libev-devel libcurl-devel zlib-devel \
+   vim-common
 
-Compiling with CMake
---------------------
+Compiling with CMake3
```

Comment:
I think the package and binary are called `cmake3` only in distributions that have cmake 2.x as default. In newer ones, it is called cmake

Reply:
I've added a note that explains how to figure out whether to call `cmake3` or `cmake`